testsuite/tests/conftest.py

```diff
@@ -250,20 +253,26 @@ def module_label(label):
     return randomize(label)
 
 
+@pytest.fixture(scope="session")
+def kuadrant_client(testconfig) -> OpenShiftClient:
+    """Returns OpenShift client for Kuadrant namespace"""
+    return testconfig["service_protection"]["system_project"]
+
+
 @pytest.fixture(scope="module")
-def kuadrant(request, testconfig, openshift):
+def kuadrant(request, testconfig, kuadrant_client):
     """Returns Kuadrant instance if exists, or None"""
     if request.config.getoption("--standalone"):
         return None
 
-    # Try if Kuadrant is deployed
-    kuadrant_openshift = testconfig["service_protection"]["system_project"]
-    kuadrants = kuadrant_openshift.do_action("get", "kuadrant", "-o", "json", parse_output=True)
-    if len(kuadrants.model["items"]) == 0:
+    try:
+        with kuadrant_client.context:
+            kuadrant = selector("kuadrant/kuadrant-sample").object(cls=KuadrantCR)
```

Comment:
Name should be either configurable, or shouldnt be needed at all, because we know there will always be only one instance of KuadrantCR